### PR TITLE
[ADD] /send command

### LINKED ISSUE #7: Add a /send command
Add a /send command that send monney to another user

### PR DESCRIPTION
**Pull request by JaavLex**
Fix #7

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -103,31 +107,58 @@ bot.on([`/${commands.list[3].name}`], (msg) => {
...
+      users[msg.chat.id][msg.from.id].wallet -= parseInt(amount)
+
+      fs.writeFile('./users_data.json', JSON.stringify(users, null, 2), 'utf8', function(err) {
+        if (err) {
+          return console.log(err)
+        }
+      })
+      msg.reply.text(`@${users[msg.chat.id][msg.from.id].username} gave ${amount} CHF to @${users[msg.chat.id][userid].username}`)
+    } catch (err) {
+      msg.reply.text(`${user} is not a user in your group/chat!`)
```
Comment (SaphireVert) [inline index.js]: Missing "@" sign

File: index.js [hunk 2]
```diff
@@ -103,31 +107,58 @@ bot.on([`/${commands.list[3].name}`], (msg) => {
...
+    }
+  } else {
+    msg.reply.text(`${amount} is an insufficient/invalid amount`)
+  }
+
+
+
```
Comment (SaphireVert) [inline index.js]: useless empty lines

**Review by nicolasreymond — APPROVED** [review]
LGTM

**Review by SaphireVert — CHANGES_REQUESTED** [review]
Need to add "price_data" file in .gitignore file and remove it from git

**Comment by ponsfrilus** [comment]
It would have been nicer to have at least 2 commits (see [Atomic Commits](https://en.wikipedia.org/wiki/Atomic_commit)): one for the tab/space hell and the other for the new feature....

**Review by SaphireVert — APPROVED** [review]
LGTM!

**Pull request merged**